test-server.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var express = require('express');
+var app = express();
+var adapter_manager = require('./adapters.js');
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+const ROOT = '/things';
```

Comment:
Using test-server.js and app.js at the same time isn't intended. I run test-server.js just as a test for exercising the adapter stuff. As soon as app.js can do get/set etc then I plan on removing test-server.js